### PR TITLE
Added README.MD and Made website Live

### PR DESCRIPTION
**Pull request by developerwaleed**
###Added README.MD and Made the website Live
>Deployed the website using Github Pages.
>The link is given in README.MD (Live Demo Link)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ Portfolio:
 MileStone 1:
 >Portfolio for Mobile Version.
 >Its responsive
+>The units like rem, em, vh ,vw are used.
```
Comment (ShadyShawkat) [inline README.md:7]: - [ ] Kindly make sure that the live demo link works and redirects the user to the demo page. Currently, it's not working.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @caasperr, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (developerwaleed) [reply README.md:7]: It is working Bother i just checked @ShadyShawkat

Comment (ShadyShawkat) [reply README.md:7]: The link in the README goes to this link (https://github.com/caasperr/Portfolio/blob/Deploy-Website/caasperr.github.io) which is not the correct link.

Comment (developerwaleed) [reply README.md:7]: i am sorry i pushed the actual link to main branch now can you check again please? @ShadyShawkat

Comment (ShadyShawkat) [reply README.md:7]: It is working fine now @caasperr 👏🏻. Kindly resubmit your project.

Comment (developerwaleed) [reply README.md:7]: I did that Thankyou soo much @ShadyShawkat

**Review by gemmen29 — APPROVED** [review]
Hi @caasperr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**